### ISSUE TITLE
two-phase PUSH

### ISSUE THREAD

**Issue by yegor256** [zold-io/zold#653]
Let's make our PUSH operation two-phases. First, we send the wallet content over HTTP PUT request. The node returns a hash code of the operation (just some random number). Then, the client has to send that code back in another PUT request. Only after receiving that code the node should consider the wallet delivered and send it to the queue for processing.

**Comment by 0crat** [comment]
Job [#653](https://github.com/zold-io/zold/issues/653) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
don't see a reason for this anymore

**Comment by 0crat** [comment]
Job `gh:zold-io/zold#653` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#653](https://github.com/zold-io/zold/issues/653) is now out of scope
